packages/isaacscript-spell/dictionaries/isaac/api.txt

```diff
@@ -1,5 +1,5 @@
-# These are words from the Lua API. Words from misspelled enums are not included, since those are
 # fixed in the reimplemented IsaacScript enums.
+# These are words from the Lua API. Words from misspelled enums are not included, since those are
```

Comment:
what is your IDE doing to mess up this file? it happens in the previous pull request too, please revert